guides/rails_guides/markdown.rb

```diff
@@ -185,7 +185,7 @@ def node_index(hierarchy)
         case hierarchy.size
         when 1
           @index_counter[2] = @index_counter[3] = @index_counter[4] = 0
-          "#{@index_counter[1] += 1}"
+          "#{@index_counter[1] += 1}."
```

Comment:
What do you think adding dot for all node indexes?
Looks like there is room for improvement in the "3.1 Prerequisites" as well.

<img width="675" alt="image" src="https://github.com/user-attachments/assets/f5bf1d8f-8cc9-4e2f-a387-dd66c2fdd0c6" />

Reply:
Now that you mentioned it, I just realized that it's okay to add `.` to the headings of the nested list. (Somehow I was under an impression of `1., 1.1, 1.2, 2.` is the common way to write the list.

<img width="838" alt="image" src="https://github.com/user-attachments/assets/2eb69e58-9d92-4944-8fc9-01dce36964a3" />

So yes, I'll add it to all node indexes.